ProjectLighthouse.Servers.Website/Pages/Layouts/BaseLayout.cshtml

```diff
@@ -191,12 +191,22 @@
     <footer>
         <div class="ui black attached inverted segment">
             <div class="ui container">
-
-                <p>@Model.Translate(BaseLayoutStrings.GeneratedBy, VersionHelper.FullVersion)</p>
+                <span>
+                    @Model.Translate(BaseLayoutStrings.GeneratedBy, VersionHelper.FullVersion)
+                </span>
                 @if (VersionHelper.IsDirty)
                 {
                     <p>@Model.Translate(BaseLayoutStrings.GeneratedModified)</p>
                 }
+                <span style="float: right;">
+                    <a href="https://github.com/LBPUnion/ProjectLighthouse/blob/main/LICENSE" target="_blank">
+                        AGPL-3.0 License
+                    </a>
+                    &centerdot;
+                    <a href="@VersionHelper.Remotes[0].Split(".git")[0]" target="_blank">
```

Comment:
Is there a chance this could throw an error if it were built improperly? I think this could be cleaned up a little to use more linq instead of array indexes. Also, have you checked that this functions properly when there is more than 1 remote?